types/html-minifier/index.d.ts

```diff
@@ -1,6 +1,7 @@
-// Type definitions for HTMLMinifier v1.1.1
+// Type definitions for HTMLMinifier v3.5.7
```

Comment:
Could you back up the old version to a folder per the readme: https://github.com/DefinitelyTyped/DefinitelyTyped#i-want-to-update-a-package-to-a-new-major-version